public/app/plugins/panel/graph3/migrations.ts

```diff
@@ -0,0 +1,266 @@
+  //     "fill": 3,
+  //     "nullPointMode": "null as zero",
+  //     "lines": true,
+  //     "linewidth": 2
+  //   }
+  // ],
+  if (angular.seriesOverrides?.length) {
+    for (const seriesOverride of angular.seriesOverrides) {
+      if (!seriesOverride.alias) {
+        continue; // the matcher config
+      }
+      const rule: ConfigOverrideRule = {
+        matcher: {
+          id: FieldMatcherID.byName,
+          options: seriesOverride.alias,
+        },
+        properties: [],
+      };
+      for (const p of Object.keys(seriesOverride)) {
+        const v = seriesOverride[p];
+        switch (p) {
+          // Ignore
+          case 'alias':
+          case '$$hashKey':
+            break;
+          // Link to y axis settings
+          case 'yaxis':
+            if (2 === v) {
+              const y2 = getFieldConfigFromOldAxis(yaxes[1]);
+              fillY2DynamicValues(y1, y2, rule.properties);
+            }
+            break;
+          case 'fill':
+            rule.properties.push({
+              id: 'custom.fillOpacity',
+              value: v / 10.0, // was 0-10
+            });
+            break;
+          case 'points':
+            rule.properties.push({
+              id: 'custom.showPoints',
+              value: v ? PointVisibility.Always : PointVisibility.Never,
+            });
+            break;
+          case 'bars':
+            if (v) {
+              rule.properties.push({
+                id: 'custom.drawStyle',
+                value: DrawStyle.Bars,
+              });
+              rule.properties.push({
+                id: 'custom.fillOpacity',
+                value: 1, // solid bars
+              });
+            } else {
+              rule.properties.push({
+                id: 'custom.drawStyle',
+                value: DrawStyle.Line, // Change from bars
+              });
+            }
+            break;
+          case 'lineWidth':
+            rule.properties.push({
+              id: 'custom.lineWidth',
+              value: v,
+            });
+            break;
+          case 'pointradius':
+            rule.properties.push({
+              id: 'custom.pointSize',
+              value: v,
+            });
+            break;
+          default:
+            console.log('Ignore override migration:', seriesOverride.alias, p, v);
+        }
+      }
+      if (rule.properties.length) {
+        overrides.push(rule);
+      }
+    }
+  }
+
+  const graph = y1.custom ?? ({} as GraphFieldConfig);
+  graph.drawStyle = angular.bars ? DrawStyle.Bars : angular.lines ? DrawStyle.Line : DrawStyle.Points;
+  if (angular.points) {
+    graph.showPoints = PointVisibility.Always;
+  } else if (graph.drawStyle !== DrawStyle.Points) {
+    graph.showPoints = PointVisibility.Never;
+  }
+  if (graph.drawStyle === DrawStyle.Bars) {
+    graph.fillOpacity = 1.0; // bars were always
+  }
+
+  graph.lineWidth = angular.lineWidth;
+  graph.pointSize = angular.pointradius;
```

Comment:
experimentally, this appears to need:

```js
graph.pointSize = 2 + angular.pointradius * 2;
```

but this falls outside the 1-10 range of the GraphNG `pointSize` slider. nevertheless, this mapping works and looks close. @torkelo should we detach `pointSize` from actual pixel diameter and find an appropriate scaling factor for a useful 1-10 range? e.g. 1px dia is pretty much useless and invisible over the thinnest available lineWidth.

Reply:
yea I think we should do something as the increments are too small to be noticable now.